sources/PTYMouseHandler.m

```diff
@@ -1271,6 +1271,12 @@ - (MouseButtonNumber)mouseReportingButtonNumberForEvent:(NSEvent *)event {
                     return MOUSE_BUTTON_BACKWARD;
                 case 4:
                     return MOUSE_BUTTON_FORWARD;
+                case 5:
+                    return MOUSE_BUTTON_10;
+                case 6:
+                    return MOUSE_BUTTON_11;
+                default:
+                    return MOUSE_BUTTON_UNKNOWN;
```

Comment:
Maybe `MOUSE_BUTTON_NONE` would be better here because then we could still report movement while an unrecognized button is pressed.

Reply:
The [xterm specs](https://invisible-island.net/xterm/ctlseqs/ctlseqs.html#h3-Extended-coordinates:~:text=the%0AalternateScroll%20resource.-,Other%20buttons,-Some%20wheel%20mice) don't support buttons beyond 11 :/
`MOUSE_BUTTON_NONE` would create confusion for applications listening to it since it corresponds to button release in (1000) mode. I think unrecognized buttons should be ignored.